### ISSUE TITLE
Namespace prefix `dapt` should not map to a metadata fragment id

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/dapt#22]
If we need a metadata only prefix then we should include something to hint at that, such as an `m`, e.g. `daptm`. That leaves us clear to define any DAPT-specific vocabulary in a `dapt` prefixed namespace without the `#metadata` fragment ID if we ever need to.

**Comment by cconcolato** [comment]
I was just following the convention at https://www.w3.org/TR/ttml-imsc1.1/#namespaces. If we think this is no longer the best approach, I'm happy to follow the recommendation.

**Comment by nigelmegitt** [comment]
Yes, that's a good convention - I think the issue is that we area actually not following it. If we were, then we would call it DAPT Metadata and use a prefix like `daptm` or whatever.

**Comment by cconcolato** [comment]
Oh, I see. The text is currently buggy. I'm fine with calling it "DAPT Metadata" and using `daptm` as prefix.
